src/magic.cpp

```diff
@@ -3706,14 +3706,24 @@ int mag_area(int skill, CharData *ch, int spellnum, int savetype) {
             continue;
 
         found = true;
-        if (damage == true)
-            mag_damage(skill, ch, tch, spellnum, savetype);
-        else {
+        if (damage == true) {
```

Comment:
You don't need the `== true`.  It's redundant.  :P

Reply:
Yes we do.  Not every spell that passes through this function deals damage.

Reply:
I mean, instead of
```if (damage == true)```
you can do
```if (damage)```